spec/lib/bank_account/money_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'bank_account'
+
+describe Bank::MoneyMarketAccount do
+  context "attributes" do
+    let(:market) {Bank::MoneyMarketAccount.new("momoney", 30000)}
+
+    it "responds to :id" do
+      expect(market).to respond_to :id
+    end
+
+    it "resonds to :balance" do
+      expect(market).to respond_to :balance
+    end
+
+    it "has an initial balance of 30000" do
+      expect(market.balance).to eq(30000)
+    end
+
+    context "low balance errors" do
+      it "raises 'Insufficient Funds' for negative intial balance" do
+        expect{Bank::MoneyMarketAccount.new("ballin", -10)}.to raise_error(ArgumentError)
+      end
+
+      it "returns 'ACCOUNT FROZEN' for balance below 10000" do
+        expect(market.withdraw(25000)).to eq("ACCOUNT FROZEN: Must make a deposit.")
+        expect(market.balance).to eq(4900)
+      end
+
+    end
+  end
+end
```

Comment:
I think you could use more tests for your edge cases. Try thinking of examples that might break your code for each requirement. If you phrase your `it` statements as behaviors, it might easier to see where you could use more tests. Everywhere you have a conditional in your code, you should have tests covering each possible outcome.

Also, when you're using inheritance, you can get away with not retesting things tested in the base class, unless you change functionality.